eos-prune-printers.service

```diff
@@ -1,8 +1,7 @@
 [Unit]
 Description=Endless prune printers
 After=cups.service
-Requires=cups.service
```

Comment:
Instead of removing this entirely, can we just change it to `Wants`? That will still pull in cups so that `lpadmin` works, but it won't actually stop this service when cups is stopped.

Reply:
Tested with Wants.  It works.
And, place the reference https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=